src/DeepEquals.js

```diff
@@ -0,0 +1,65 @@
+// Patched version of fast-deep-equal which does not
+// allocate memory via calling Object.keys
```

Comment:
Seems polite to leave a link to https://github.com/epoberezkin/fast-deep-equal/blob/master/index.js